### PR TITLE
use target include directories

### PR DESCRIPTION
**Pull request by Karsten1987**
as per comment. Same as https://github.com/ros-perception/laser_geometry/issues/53

Signed-off-by: Karsten Knese <karsten@openrobotics.org>

### CODE REVIEW

**Comment by Karsten1987** [comment]
as discussed, changing the base branch back to `ros2` as this is targeted for foxy.

**Comment by chapulina** [comment]
Foxy CI: [Build Status](https://ci.ros2.org/job/ci_linux/9255/)

File: gazebo_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -198,6 +198,7 @@ ament_export_libraries(gazebo_ros_tricycle_drive)
 add_library(gazebo_ros_video SHARED
   src/gazebo_ros_video.cpp
 )
+target_include_directories(gazebo_ros_video PUBLIC include)
```
Comment (chapulina) [inline gazebo_plugins/CMakeLists.txt:201]: `gazebo_ros_video` is not being able to find `image_transport` includes on CI:

```
19:44:23 [79.272s] [K/home/jenkins-agent/workspace/ci_linux/ws/src/Karsten1987/gazebo_ros_pkgs/gazebo_plugins/src/gazebo_ros_video.cpp:26:[Kfatal erro[Kimage_transport/image_transport.h: No such file or directory
19:44:23 [79.272s]  #include [K<image_transport/image_transport[K
19:44:23 [79.272s]           [K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[K
19:44:23 [79.272s] compilation terminated.
```

Comment (chapulina) [reply gazebo_plugins/CMakeLists.txt:201]: @Karsten1987, did you have a chance to look into this?

Comment (Karsten1987) [reply gazebo_plugins/CMakeLists.txt:201]: I did not, but the fix should be trivial; Essentially adding "image_transport" to the list of `ament_target_dependencies`

**Comment by chapulina** [comment]
@ros-pull-request-builder retest this please

**Comment by chapulina** [comment]
@ros-pull-request-builder retest this please

**Review by chapulina — APPROVED** [review]
Compiles fine, the test failures are already happening on `ros2`.

**Pull request merged**